plugins/cody-chat/src/com/sourcegraph/cody/CodyPaths.java

```diff
@@ -27,6 +27,15 @@ public static ProjectDirectories projectDirectories() {
     return ProjectDirectories.from("com.sourcegraph", "Sourcegraph", "CodyEclipse");
   }
 
+  // The project directory that the cody app (agent) uses for its config files
```

Comment:
We should probably allow clients to determine this path instead of having the server default to a location.

Reply:
Just thinking out loud, I'm not loving how we guess the path here.